examples/timestamping/backend/src/lib.rs

```diff
@@ -52,28 +54,31 @@ use crate::{
 };
 
 #[derive(Debug, ServiceFactory)]
-#[exonum(proto_sources = "proto", implements("TimestampingInterface"))]
+#[exonum(
+    proto_sources = "proto",
+    implements("TimestampingInterface", "Initialize")
```

Comment:
@alekseysidorov Interestingly, the following changes seem to work without any changes to proc macros:

```rust
pub trait Initialize {
    type Params: BinaryValue;
    fn initialize(&self, context: TransactionContext, params: Self::Params)
        -> Result<(), ExecutionError>;
}

impl<T: BinaryValue> Interface for dyn Initialize<Params = T> {
    const INTERFACE_NAME: &'static str = "Initialize";
    // probably, this constant needs to be exposed separately,
    // because call sites looks awkward otherwise.
    
    // snip
}

#[derive(Debug, ServiceFactory)]
#[exonum(
    // snip
    implements("TestService", "Initialize<Params = Init>")
)]
pub struct TestServiceImpl;

impl Initialize for TestDispatcherService {
    type Params = TestExecute;

    fn initialize(
        &self,
        _context: TransactionContext,
        params: TestExecute,
    ) -> Result<(), ExecutionError> {
        // snip
    }
}
```